tests/post-checkout.bats

```diff
@@ -2,62 +2,97 @@
 
 # load '/usr/local/lib/bats/load.bash'
 load "${BATS_PATH:?}/load.bash"
+source "/plugin/lib/common.sh"
+
+# Create fake "buildkite-agent"
+cp /plugin/tests/fake-buildkite-agent.sh /usr/local/bin/buildkite-agent
+chmod +x /usr/local/bin/buildkite-agent
```

Comment:
Should these three lines (lines 7 through 9) go into the `setup()` function instead?

Reply:
There's no need; installing it once works for all tests.